tock/api/tests.py

```diff
@@ -39,11 +46,17 @@ def test_projects_csv(self):
 class ProjectInstanceAPITests(WebTest):
     fixtures = FIXTURES
 
+    def setUp(self):
+        self.user = User.objects.get(username='aaron.snow')
+        self.token = Token.objects.get_or_create(user=self.user)[0].key
+        self.client = APIClient()
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token)
+
```

Comment:
We're duplicating this setUp code here in several places. It probably makes sense to create a function that the various setUp functions all call, so that if we need to update one of these setup steps, we only have to update it in one place.